javascript/chronometer.js

```diff
@@ -1,39 +1,62 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+  
+    this.currentTime = 0
+    this.intervalId = null
   }
 
   start(callback) {
-    // ... your code goes here
+   
+    this.currentTime = 0;
+    this.intervalId = setInterval(() => {
+      this.currentTime++;
+      return this.currentTime;
```

Comment:
Aqui hizo falta verificar si nos estan pasando el callback y ejecutarlo si lo recibimos👀 
```suggestion
      return this.currentTime;
      if (callback) callback();
```